source/crd.go

```diff
@@ -116,7 +118,15 @@ func (cs *crdSource) AddEventHandler(handler func() error, stopChan <-chan struc
 func (cs *crdSource) Endpoints() ([]*endpoint.Endpoint, error) {
 	endpoints := []*endpoint.Endpoint{}
 
-	result, err := cs.List(&metav1.ListOptions{})
+	var result *endpoint.DNSEndpointList
```

Comment:
```go
var (
  result *endpoint.DNSEndpointList
  err error
)
```

Reply:
It looks like this comment may have been missed

Reply:
Sorry, I must have removed it one time during a rebase. Should be good now